pennylane/labs/resource_estimation/__init__.py

```diff
@@ -35,7 +35,10 @@
     ~GateCount
 
 Resource Estimation Functions:
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+WARNING: Failed to import pennylane.labs.
+Possible hints:
+* ImportError: cannot import name 'ResourceOperator' from partially initialized module 'pennylane.labs.resource_estimation' (most likely due to a circular import) (/Users/diksha.dhawan/Documents/resource_estimation/pennylane_review/pennylane/labs/resource_estimation/__init__.py)
+* AttributeError: module 'pennylane' has no attribute 'labs'~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
Copy paste typo probably? 
```suggestion
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```